python/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+if(WIN32 AND CMAKE_BUILD_TYPE STREQUAL "Debug")
+  # pybind11 logic for setting up a debug build when both a debug and release
+  # python interpreter are present in the system seems to be pretty much broken.
+  # This works around the issue.
+  set(PYTHON_LIBRARIES "${PYTHON_DEBUG_LIBRARIES}")
+endif()
+
+
+if(USE_SYSTEM_PATHS_FOR_PYTHON_INSTALLATION)
+  if(${CMAKE_VERSION} VERSION_LESS "3.12.0")
+    execute_process(
+      COMMAND "${PYTHON_EXECUTABLE}" -c "if True:
+  from distutils import sysconfig as sc
+  print(sc.get_python_lib(plat_specific=True))"
+      OUTPUT_VARIABLE Python3_SITEARCH
+      OUTPUT_STRIP_TRAILING_WHITESPACE)
+  else()
+    # Get install variable from Python3 module
+    # Python3_SITEARCH is available from 3.12 on, workaround if needed:
+    find_package(Python3 COMPONENTS Interpreter)
+  endif()
+
+  if(USE_DIST_PACKAGES_FOR_PYTHON)
+    string(REPLACE "site-packages" "dist-packages" IGN_PYTHON_INSTALL_PATH ${Python3_SITEARCH})
+  endif()
+else()
+  # If not a system installation, respect local paths
+  set(IGN_PYTHON_INSTALL_PATH ${IGN_LIB_INSTALL_DIR}/python)
+endif()
+
+# Set the build location and install location for a CPython extension
+function(configure_build_install_location _library_name)
+  # Install library for actual use
+  install(TARGETS ${_library_name}
+    DESTINATION "${IGN_PYTHON_INSTALL_PATH}"
+  )
+endfunction()
+
+pybind11_add_module(sdformat SHARED
+  src/sdf/_ignition_sdformat_pybind11.cc
+)
+
+target_link_libraries(sdformat PRIVATE
+  ${PROJECT_LIBRARY_TARGET_NAME}
+)
+
+configure_build_install_location(sdformat)
+
+if (BUILD_TESTING)
+  set(python_tests
+  )
+
+  foreach (test ${python_tests})
+    add_test(NAME ${test}.py COMMAND
+      "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/python/test/${test}.py")
```

Comment:
Should we use `Python3_EXECUTABLE` instead? I think `PYTHON_EXECUTABLE` is 
deprecated and we now use `find_package(Python3)` in ign-cmake as of
https://github.com/ignitionrobotics/ign-cmake/pull/218